packages/perseus/src/widgets/numeric-input.jsx

```diff
@@ -448,6 +448,9 @@ export class NumericInput extends React.Component<Props, State> {
             );
         }
 
+        // Note: This is _very_ similar to what `input-number.jsx` does. If
+        // you modify this, double-check if you also need to modify that
+        // component.
```

Comment:
Would it be worth circling back and creating a shared set of logic?

Reply:
I'd rather we work to eliminate this widget. See: LC-532